lints.toml

```diff
@@ -46,6 +46,7 @@ allow = [
     'clippy::shadow_same',
     'clippy::unreachable',
     'clippy::use_self',
+    'clippy::type_repetition_in_bounds',
```

Comment:
I think it is ok not to allow it. If we have only one bound T: ... instead of two or three, then it is more readable.

Reply:
It breaks on our code in http

Reply:
We can allow it only in http if it is impossible to work without it there. What do you think about it?

Reply:
Is it necessary in http, can you send the place?

Reply:
```rust
async fn wrapper_handler<State, Path, Query, Req, Fut, F>(
    fut: F,
    state: State,
    path: Result<Path, Path::Error>,
    query: Result<Query, Query::Error>,
    req: Result<Req, Req::Error>,
) -> http::HttpResponse
where
    State: Clone + Send + Sync + 'static,
    Fut: Future<Output = http::HttpResponse> + Send,
    F: Send + Sync + 'static + Fn(State, Path, Query, Req) -> Fut,
    Path: TryFrom<http::PathParams> + Send + Sync + 'static,
    Path::Error: http::HttpResponseError,
    Query: TryFrom<http::QueryParams> + Send + Sync + 'static,
    Query::Error: http::HttpResponseError,
    Req: TryFrom<http::RawHttpRequest> + Send + Sync + 'static,
    Req::Error: http::HttpResponseError,
```

It doesnt like using generics 2 times here, though it is impossible to implement without it. Lint also is kinda unhelpfull, as there are no places in codebase there it can be triggered (it is easy to catch duplication of types in bounds by eye)

Reply:
It doesnt like using `Path` and `Path::Error`

Reply:
Let's allow it only in that place/module then?

Reply:
Lint also is kinda unhelpfull, as there are no places in codebase there it can be triggered (it is easy to catch duplication of types in bounds by eye)

Reply:
Dont like fashion of having lots of allows in codebase

Reply:
But it is better to catch by lint then by eyes when possible